src/config_creature.h

```diff
@@ -191,11 +185,8 @@ struct Creatures { // sizeof = 16
 };
 
 /******************************************************************************/
-DLLIMPORT struct Creatures _DK_creatures[CREATURE_TYPES_COUNT];
-#define creatures _DK_creatures
-DLLIMPORT unsigned short _DK_breed_activities[CREATURE_TYPES_COUNT];
-#define breed_activities _DK_breed_activities
-
+extern struct Creatures creatures[CREATURE_TYPES_MAX];
+extern unsigned short breed_activities[CREATURE_TYPES_MAX];
```

Comment:
I see the actual definition is in thing_creatures.c, makes more sense to have the forward declaration in the .h of the same file as where the actual data is